packages/aws-cdk/README.md

```diff
@@ -250,6 +250,9 @@ $ # Deploys only to environments foo and bar
 $ cdk bootstrap --app='node bin/main.js' foo bar
 ```
 
+By default, bootstrap stack will be protected from stack termination. This can be disabled using 
+`--termination-protection` argument.
```

Comment:
I believe yargs also supports `-no--termination-protection` as well (convention for boolean flags)